feat(sig-services): add security implications requirement (#210)
open-services-group/community: sig-services/service-requirements/security-implications.md

---
title: Define and explain the security implications
persona: service provider/owner
phase: service-design, service-operational
---

> The service should have defined security implications that are applied when a user subscribes to the service. The document that is describing these security implications should also define why are they required for the full functionality of the service.

Reviewer: I would rephrase this, security implication isn't something that "is applied" I think.. it's rother something that "happens"...

```suggestion
The service should disclose all security implications that can have an impact on a user which subscribes to the service. These implications are listed in a document which describes in detail what is the risk and why it is necessary for the full functionality of the service. In case there's an option to opt-out of a particular risk factor the document states what effect it has on service functionality.
```

Author: Agree, the opt-out option is also a good idea.

## Requirements

> - Define the security implications and explain why they are required in a form (a separate document or a prompt before subscription to the service) easily accessible by the user

Reviewer: ```suggestion
- Define and disclose all the security implications and explain why they are required in a form (a separate document or a prompt before subscription to the service) easily accessible by the user
```

## Reasoning

> Defining and explaining the security implications can help the user with assessing the security risk associated with using the service.

Reviewer: ```suggestion
Defining and explaining the security implications can help the user with assessing the security risk associated with using the service. It boosts service transparency and rationalize user expecations.
```
